src/test/resources/uk/gov/justice/digital/hmpps/prisonestate/resource/prison_id_badrequest_getPrisonFromGpPrescriber.json

```diff
@@ -0,0 +1,7 @@
+{
+  "status": 400,
+  "errorCode": null,
+  "userMessage": null,
+  "developerMessage": "getPrisonFromGpPrescriber.prisonId: size must be between 0 and 3",
```

Comment:
isn't the size limit 6 rather than 3?

Reply:
prisonId 3 and gp code 6 I think?

Reply:
agreed, that's why the build is failing too :man_facepalming: